OpenRA.Mods.Common/Activities/Transform.cs

```diff
@@ -168,11 +168,5 @@ public Order IssueOrderForTransformedActor(Actor newActor)
 		{
 			return new Order(orderString, newActor, target, true);
 		}
-
-		public override Activity Tick(Actor self)
```

Comment:
This is in the wrong commit ("Make Tick return bool." instead of "Add default for Activity-Tick").